src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -350,442 +317,10 @@ corner_radius_bottom_left = 10
 
 [sub_resource type="ButtonGroup" id=37]
 
-[sub_resource type="DynamicFont" id=38]
-size = 15
-use_filter = true
-font_data = ExtResource( 5 )
-
 [sub_resource type="StyleBoxTexture" id=39]
 texture = ExtResource( 88 )
 region_rect = Rect2( 0, 0, 258, 1 )
 
-[sub_resource type="DynamicFont" id=40]
-size = 12
-use_filter = true
-font_data = ExtResource( 55 )
-
-[sub_resource type="Theme" id=41]
```

Comment:
This theme is exclusively for lowering the size of the texts on the membrane color picker, so the left panel doesn't get too big. Might be a bit hacky, but the theming for ColorPicker itself is a bit all over the place.
![2020-12-04_23 48 31 7474](https://user-images.githubusercontent.com/54026083/101192089-212fc700-368d-11eb-9a5b-d6189b07955d.png)

Reply:
Didn't notice that. Can you / do you know the easiest way to fix that?

Reply:
Could the colour picker theme stuff be put in the main thrive theme? I think that'd be nice if that worked.

Reply:
No, as far as I know ColorPicker is composed of separate controls (like the check button, label and line edit). In thrive theme their styling have to be then tweaked just for the color picker's specific use case (which also affect styling on other controls using thrive theme), so I think a separate theme have to be used on it.

Reply:
Okay, that's unfortunate. 
Just had another idea, I think it should be possible to make a TweakedColourPicker.tscn file that just has a colour picker in it and a theme, though I suppose for passing through signals it might need a short C# script file to do that. I can make a separate issue to fix this at some point in order to not have to do tweaks in multiple places (when we get more than one colour picker in the GUI)

Reply:
Ah yeah, that seems good.